apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/behandling/Behandling.kt

```diff
@@ -83,8 +84,8 @@ data class Beregningsperiode(
 data class Persongalleri(
     val innsender: Innsender,
     val soeker: Soeker,
-    // TODO: Legge til gjenlevende, men det krever litt logikk siden gjenlevende er null på OMS (søker = gjenlevende)
-    val avdoed: Avdoed
+    val avdoed: Avdoed,
```

Comment:
Hva skjer når det er to avdøde, slik som det kanskje er i en slik sak med verge?

Reply:
P.t. så er det ingen del av verdikjeden vår som støtter det. Vi må først legge til rette for at Grunnlag støtter det før vi kan tenke på det i andre apper.